tests/palindrome_number_test.py

```diff
@@ -1,2 +1,35 @@
 import pytest
 from functions.palindrome_number import palindrome_number
+
+
+def test_returns_true_for_99():
+    # Arrange
+    n = 99
+
+    # Act
+    answer = palindrome_number(99)
+
+    # Assert
+    assert answer == True
+
+
+def test_returns_true_for_151():
```

Comment:
I think adding an edge case will be good here! (But don't really do it)

Reply:
Ok, I won't do it! :)